db/write_batch_test.cc

```diff
@@ -899,6 +904,89 @@ TEST_F(WriteBatchTest, MemoryLimitTest) {
   ASSERT_TRUE(s.IsMemoryLimit());
 }
 
+TEST_F(WriteBatchTest, AssignTimestamps) {
+  // We assume the last eight bytes of each key is reserved for timestamps.
+  // Therefore, we must make sure each key is longer than eight bytes.
+  constexpr size_t key_size = 16;
+  constexpr size_t num_of_keys = 10;
+  std::vector<std::string> key_strs;
+  for (size_t i = 0; i < num_of_keys; ++i) {
+    key_strs.emplace_back(std::string(key_size, '\0'));
+  }
```

Comment:
All keys are meant to be the same (i.e. 16 `'\0'` characters), correct? If so, we could initialize the `vector` the same way we do the `string`, i.e. by passing the number of elements and the element to use.

Reply:
Done.